libs/openchallenges/api-description/src/components/schemas/ChallengePlatformName.yaml

```diff
@@ -1,4 +1,17 @@
 description: The name of the challenge platform.
 type: string
-minLength: 3
-maxLength: 30
+enum:
+  - Synapse
+  - CAMI
+  - Grand Challenge
+  - precisionFDA
+  - Kaggle
+  - CodaLab
+  - CodaBench
+  - Eterna
+  - Nightingale OS
+  - EvalAI
+  - CACHE
+  - CAMEO
+  - DrivenData
+example: Synapse
```

Comment:
We don't want an enum as platform data live in the DB, not in the API description.

Reply:
I have remove the enum values in descriptions